pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -145,7 +148,7 @@ func (c *Cluster) makeSetServerAddrInitContainer(mgrConfig *mgrConfig, mgrModule
 			keyring.VolumeMount().Admin(),
 		),
 		Env: append(
-			opspec.DaemonEnvVars(c.cephVersion.Image),
```

Comment:
@BlaineEXE to make the unit tests work with your suggested way we need to have the additional vars in the init containers as well.